cvat-ui/src/components/layout-grid/LayoutGrid.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
Could you please follow the existing way of naming files? e.g. ``layout-grid.tsx``

Reply:
Sure